modules/@ergo/core/models/stringWrapper.js

```diff
@@ -45,3 +45,23 @@ export function capitalizeAndConcatenationArray(strings) {
 export function getUUID() {
     return Math.random().toString(36).substr(2, 9);
 }
+
+
+/**
+ * Returns formatted bytes string
+ * @function
+ * @param {string} bytes
+ * @param {number} decimal places
+ * @returns {string} formatted bytes string
+ */
+export function formatBytes(bytes, decimals = 2) {
+    if (bytes === '0') return '0 Bytes';
+
+    const k = 1024;
+    const dm = decimals < 0 ? 0 : decimals;
+    const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
```

Comment:
to 'TB' is enough  I think  :)

Reply:
Just in case to sleep peacefully xD It won't affect any performance drop